src/utilities/qstring.c

```diff
@@ -388,10 +393,8 @@ char *qstrdup_between(const char *str, const char *start, const char *end) {
 
     int len = e - s;
 
-    char *buf = (char *) malloc(sizeof(char) * (len + 1));
-    strncpy(buf, s, len);
+    char *buf = qmemdup(s, len);
```

Comment:
doesn't it need to be `len + 1` and also need NULL check for the buf?
Let's focus on the NULL return. Please revert this.

Reply:
[Fixed](https://github.com/wolkykim/qlibc/pull/105/commits/701e136a03cfd074aa52c8efae8c15721844115a).